superset/exceptions.py

```diff
@@ -270,3 +270,7 @@ class SupersetCancelQueryException(SupersetException):
 
 class QueryNotFoundException(SupersetException):
     status = 404
+
+
+class ColumnNotFoundException(SupersetException):
```

Comment:
Should we make this a 404/not found status?

Reply:
Done